packages/esm-appointments-app/src/appointments/common-components/appointments-table.component.tsx

```diff
@@ -89,8 +88,8 @@ const AppointmentsTable: React.FC<AppointmentsTableProps> = ({ appointments, isL
       key: 'status',
     },
   ];
-
-  const rowData = results?.map((appointment) => ({
+  const activeAppointments = results?.filter((appointment) => appointment.status !== 'Cancelled');
+  const rowData = activeAppointments?.map((appointment) => ({
```

Comment:
The problem with doing this is that this appointment-table component is used in multiple places, and is meant just to display all the appointments passed into it.  For example, it is used to render the table that shows the view of "Cancelled" appointments, so this would no longer work with this change.

The appointment-list component is probably the best place to handle this... it takes in a "status" prop and fetches all appointments with the specified status, and then passes that list in as a prop to the appointment-list.

If the status prop passed into appointments-list is empty, it just loads all appointments.

One way to get this to work could be to the add a prop "filterCancelled" to appointment-list component.  this prop should be optional and default to false, but, if true, would apply the above filter before passing the appointments into the appointments-table.

Reply:
@mogoodrich Thanks for the review. Cancelled appointments are now filtered only in home dashboard